release.csx

```diff
@@ -0,0 +1,84 @@
+#r ".\packages\Octokit.0.26.0\lib\net45\Octokit.dll"
+#r ".\packages\system.runtime.interopservices.runtimeinformation\4.0.0\lib\net45\System.Runtime.InteropServices.RuntimeInformation.dll"
+
+using Octokit;
+
+const string repoOwner = "FakeItEasy";
+const string repoName = "FakeItEasy";
+const string remote = "upstream";

```

Comment:
Not everyone calls their upstream remote `upstream`. We could use the URL instead.

Reply:
Yes. Good idea. What do you call yours?

Reply:
`upstream`. But it doesn't mean everyone does ;)